agent/spec/lib/kontena/models/service_pod_spec.rb

```diff
@@ -345,8 +353,9 @@
     it 'returns correct array when volumes are defined' do
       data['volumes'] = volumes
       bind_vols = subject.build_bind_volumes
-      expect(bind_vols.size).to eq(1)
-      expect(bind_vols[0]).to eq('/proc:/host/proc:ro')
+      expect(bind_vols.size).to eq(2)
+      expect(bind_vols[0]).to eq('app.someVol:/data:')
```

Comment:
yes, I actually tested it manually. :) But now a big issue to take out in code for clarity.